core/protocols/aws-json-protocol/src/main/java/software/amazon/awssdk/protocols/json/BaseAwsJsonProtocolFactory.java

```diff
@@ -55,7 +55,7 @@ public abstract class BaseAwsJsonProtocolFactory {
     /**
      * Used by operations that do not serialize the input, e.g., when the input is not defined in the model. RPCv2 uses it.
      */
-    public static final OperationMetadataAttribute<Boolean> USE_NO_OP_GENERATOR = new OperationMetadataAttribute<>(Boolean.class);
+    public static final OperationMetadataAttribute<Boolean> GENERATES_BODY = new OperationMetadataAttribute<>(Boolean.class);
```

Comment:
This is protected API, so we should probably honor backwards compatibility

Reply:
> This is protected API, so we should probably honor backwards compatibility

This attribute has not yet been published and was introduced in the past PR. Just changing the name to clarify its intent.